src/lib.rs.in

```diff
@@ -852,6 +853,10 @@ foreigner_class!(
 class IronOxide {
     self_type IronOxide;
     private constructor = empty;
+    /// Clears all entries from the policy cache
+    /// 
+    /// @return number of entries cleared from the cache
+    method BlockingIronOxide::clear_policy_cache(&self) -> usize; alias clearPolicyCache;
```

Comment:
I think this should be 
``` method IronOxide::clear_policy_cache(&self) -> usize; alias clearPolicyCache;```
to remain consistent with the rename of `BlockingIronOxide` -> `IronOxide` that occurrs in `lib.rs`

Reply:
Changed